lib/App.js

```diff
@@ -247,20 +285,34 @@ module.exports = class App {
 		this.reset();
 	}
 
+	/**
+	 * @param {any} message 
+	 * @param {string[]} args 
+	 */
 	onSetGameMode(message, args) {
 		// Check if a game is already running
-		if (this.gameState !== GameStateEnum.NOTRUNNING) {
+		if (this.gameState !== GameStateEnum.NOT_RUNNING) {
 			message.channel.send(`Es ist bereits ein Spiel aktiv`);
 			return;
 		}
 
-		if (args.length >= 2 && Object.values(GameModeEnum).indexOf(args[1]) > -1) {
-			this.gameMode = args[1];
-			message.channel.send(`Der Spielmodus wurde angepasst auf ${ this.gameMode }`);
+		if (args.length < 2) {
+			message.channel.send(`Falsche Anzahl an Argumenten! ${ USE_HELP_COMMAND }`);
 			return;
 		}
 
-		message.channel.send(`Der Spielmodus wurde nicht gefunden`);
+		const newGameMode = args[1];
```

Comment:
Please use let inside functions. Even if it wont change anymore. See other const too.